lib/fastlane/plugin/fueled/actions/generate_files_flutter.rb

```diff
@@ -30,7 +32,15 @@ def self.available_options
             description: "The build variant used for generating config file",
             is_string: true,
             default_value: "debug"
-          )
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :skip_localization,
+            env_name: "SKIP_LOCALIZATION",
+            description: "Whether to skip generating the localization files",
+            optional: true,
+            is_string: false,
+            default_value: true
```

Comment:
Shouldn't we make the default value false? we should ideally try to always ensure strings are localized with the library and the off case is they're not rather than the other way round.

Reply:
I actually agree with @humblerookie here. There are some things we want to make it part of our process and ensure that we do it in all our projects. Support for Localization, accessibility and Unit Testing are definitely at the top of the list.

Reply:
Alright, I will change it.